main.rb

```diff
@@ -0,0 +1,576 @@
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
```

Comment:
__[rubocop]__
* Method `Object#baz` is defined at both /sideci/sandbox2/main.rb:9 and /sideci/sandbox2/main.rb:273.
* Use empty lines between method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods)